changelogs/unreleased/3470-unserializable-items-in-the-changes-set-make-the-CRUD-handler-get-stuck-in-updating.yml

```
@@ -0,0 +1,7 @@
+description: Cast items in the 'changes' set that aren't JSON-serializable to str.
```

Keep in mind that changelog are intended to be read by end-user. A good changelog entry should describe what has been fixed without going in depth about the implementation details.
```suggestion
description: Fix bug that fails the CRUDHandler when a changed attribute is of type set.
```